android-sdk/src/main/java/com/mobilecoin/lib/BlockchainClient.java

```diff
@@ -12,30 +12,28 @@
 import com.mobilecoin.lib.util.NetworkingCall;
 
 import java.math.BigInteger;
-import java.time.Duration;
-import java.time.LocalDateTime;
 
 import consensus_common.ConsensusCommon;
 
 class BlockchainClient extends AnyClient {
     private static final String TAG = BlockchainClient.class.getName();
     private static final BigInteger DEFAULT_TX_FEE = BigInteger.valueOf(10000000000L);
-    private final Duration minimumFeeCacheTTL;
+    private final long minimumFeeCacheTTLms;
     private volatile ConsensusCommon.LastBlockInfoResponse lastBlockInfo;
-    private LocalDateTime lastBlockInfoTimestamp;
+    private long lastBlockInfoTimestampms;
```

Comment:
```suggestion
    private long lastBlockInfoTimestamp_ms;
```
nitpick but `Timestampms` looks like a typo

Reply:
Yeah, I wasn't sure what the best way to include the units in the name, but I guess that's it. I'll commit this suggestion and update the other names as well